Source/gms2.api.js

```diff
@@ -12,11 +12,11 @@ class SocketIO {
 
     connect() {
         this.socket = io();
-        this.init_socket_event();
+        this.init_socket_event({ transports: ["websocket", "polling"] });
```

Comment:
You're passing the object to the wrong function, you should pass it to io()

Reply:
What about creating "this.options = { transports: ["websocket", "polling"] }" in the constructor and reusing it in connect() and connect_to_url()?

Reply:
And thanks for the PR! :)

Reply:
> You're passing the object to the wrong function, you should pass it to io()

[Commit](https://github.com/ignoxx/SocketIO-GMS2-Extension/pull/12/commits/c53b95a35ec2c14d0a45c3acc59ccfe1e0c77acb) → Oh, sorry I fixed. Thansk

Reply:
Yea I got the reason, it makes sense to set it to use websockets first. What I had in mind was to have the transport options `{ transports: ["websocket", "polling"] }` defined once in the constructor like 

`this.options =  { transports: ["websocket", "polling"] }` 

and than being re-used in `connect()` and `connect_by_url()` by 

`io(this.options)` `io(url, this.options)`. 

So we don't repeat ourselfs and be less error prone :)

Reply:
> What I had in mind was to have the transport options { transports: ["websocket", "polling"] } defined once in the constructor like

[Commit](https://github.com/ignoxx/SocketIO-GMS2-Extension/pull/12/commits/659ade58761d4ad02c0de4a10cbcd7c41a5a3888) → OK, I Fixed it

Reply:
Awesome, thanks!